rootfs/scheduler/__init__.py

```diff
@@ -993,6 +993,10 @@ def _wait_until_pods_are_ready(self, namespace, container, labels, desired):  #
             logger.info('timed out ({}s) waiting for pods to come up in namespace {}'.format(timeout, namespace))  # noqa
 
         logger.info("{} out of {} pods in namespace {} are in service".format(count, desired, namespace))  # noqa
+        if count != desired:
+            # raising to allow operations to rollback
+            raise KubeException('Not enough pods in namespace {} came into service. '
+                                '{} out of {}'.format(namespace, count, desired))
```

Comment:
If the pods are in a crash loop back/if they are atleast started the pods logs can be anyway found using the deis logs....i just want to see the possibility if we can get the reason for pods not starting in  other scenarios may be getting it from the events.The reason we might want to show is because users may be in confused state as in why the release or scale failed.
Also i have one more question regarding this change....do we want to think the operation as failure even if just the readiness probe failed because it might be waiting for some external dependency?